### PR TITLE
new PR, update RAG HR chatbot article (re notebook/colab changes)

### PR DESCRIPTION
**Pull request by robertdhayanturner**

### CODE REVIEW

File: docs/articles/rag_hr_chatbot.md [hunk 1]
```diff
@@ -182,108 +193,133 @@ source.put([text_df])
...
+)
+
+maternity_result_df = present_result(maternity_result)
+maternity_result_df
 ```
 
-![Results 4](../assets/use_cases/rag_hr_chatbot/result4.png)
+![Results 4](../assets/use_cases/rag_hr_chatbot/results4.png)
 
 Now that we’re happy with the retrieval, let’s set up the generation.
```
Comment (robertdhayanturner) [inline docs/articles/rag_hr_chatbot.md:324]: detected from the `natural_query` terms - e.g., "recent useful info on management responsibilities"

File: docs/articles/rag_hr_chatbot.md [hunk 2]
```diff
@@ -182,108 +193,133 @@ source.put([text_df])
...
+    search_query="What are the companies terms for maternal leave?",
+    limit=TOP_N,
+)
+
+maternity_result_df = present_result(maternity_result)
+maternity_result_df
 ```
 
-![Results 4](../assets/use_cases/rag_hr_chatbot/result4.png)
+![Results 4](../assets/use_cases/rag_hr_chatbot/results4.png)
```
Comment (robertdhayanturner) [inline docs/articles/rag_hr_chatbot.md:322]: how to interpret extremely low usefulness scores of our results?

File: docs/articles/rag_hr_chatbot.md [hunk 3]
```diff
@@ -103,50 +111,49 @@ TOP_N = 10
...
-] * len(text_old)
 text_df.head()
 ```
 
+Let's take a sneak peak at the first five rows of our data, which comes from all three policy documents, and has the following columns:
+
+- index
+- body: text of the HR policy paragraph
+- creation_date: utc timestamp when the policy was issued
+- usefulness: ratio of people who (when prompted) deem the paragraph useful
```
Comment (robertdhayanturner) [inline docs/articles/rag_hr_chatbot.md:125]: people = employees? this is a human review we have access to as part of the source data?

File: docs/articles/rag_hr_chatbot.md [hunk 4]
```diff
@@ -35,60 +38,65 @@ For companies aiming to make HR documents more accessible to employees and contr
 
 ### Data sources
 
-In our hypothetical company, there are two HR policy sources:
+In our hypothetical company, there are three HR policy sources:
+
 - an older (2023) HR policy, which contains a maternal leave policy and details on manager responsibilities
-- a newer (2024) HR policy document that contains updated information about management responsibilities, alongside some other HR policy information
+- a more recent HR policy document, containing inaccurate information on management responsibilities, but also unique information about paternity leave
```
Comment (robertdhayanturner) [inline docs/articles/rag_hr_chatbot.md:44]: year?

File: docs/articles/rag_hr_chatbot.md (hunk 3 above)
Comment (morkapronczay) [reply docs/articles/rag_hr_chatbot.md:125]: yes

File: docs/articles/rag_hr_chatbot.md (hunk 4 above)
Comment (morkapronczay) [reply docs/articles/rag_hr_chatbot.md:44]: 2024, like the other fresh one

File: docs/articles/rag_hr_chatbot.md (hunk 2 above)
Comment (morkapronczay) [reply docs/articles/rag_hr_chatbot.md:322]: I don't think that is the case - around 0.6 is ok usefulness score

File: docs/articles/rag_hr_chatbot.md (hunk 1 above)
Comment (morkapronczay) [reply docs/articles/rag_hr_chatbot.md]: yes, the Natural Language Querying works like that - an LLM sets the query params based on the detected intent of the user's free text query

**Pull request merged**